### ISSUE TITLE
Feature: deeplink to Curate for users to submit entries to the registries directly

### ISSUE THREAD

**Issue by gmkung** [kleros/contract-insights-snap#27]
From what i remember, Curate allows us to deeplink to the submission form of a registry by encoding the form fields in the URL. Perhaps we can do that in the Snap, so that if the data is not present, someone can just click on the deeplink and go directly to Curate to contribute the submission? Though adding it might make the audit more problematic...
@greenlucid

**Comment by gmkung** [comment]
Hmm yea I guess this is only viable for the CDN (in which case it's the perfect opportunity as the domain and contract are there, and they can take a screenshot of the exact screen they are there as the 'proof').

**Comment by gmkung** [comment]
Or for the others we can populate what we can like address and chain, and those who want, fill in the rest. Are we able to maybe minimise this to just a mini question mark that's like a tooltip? That'd be less intrusive.

**Comment by greenlucid** [comment]
(on the mini question tool tip:) I don't think thats an option, I don't see such a component.

https://docs.metamask.io/snaps/how-to/use-custom-ui/

But, we could be using some other concepts from this, such as the loading indicator while we wait for the subgraph to respond

**Comment by gmkung** [comment]
Yeah I like the usage of the loading indicator and I like this too:

> Contract Tag: Not Found ([Tag me](https://curate.kleros.io/tcr/100/0x76944a2678A0954A610096Ee78E8CEB8d46d5922?action=submit&Public%20Name%20Tag=&Contract%20Address=0xc0fec0fede4dde4d))
